src/_gettsim_tests/test_piecewise_polynomial.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tests for `piecewise_polynomial`
+"""
+import numpy as np
+from _gettsim.piecewise_functions import (
+    get_piecewise_parameters,
+)
+
+
+def test_get_piecewise_parameters_all_intercepts_supplied():
+    params = {
+        "type": "piecewise_linear",
+        "progressionsfaktor": True,
```

Comment:
Is this required? What does it do?

Reply:
not in our case. I deleted it for simplicity reasons